newIDE/app/src/ProjectCreation/CreateProjectDialog.js

```diff
@@ -67,99 +67,189 @@ export type OnCreateFromExampleShortHeaderFunction = ({|
-          />,
-        ]}
-        secondaryActions={[
-          this.state.currentTab === 'games-showcase' ? (
-            <FlatButton
-              key="submit-game-showcase"
-              onClick={() => {
-                Window.openExternalURL(
-                  'https://docs.google.com/forms/d/e/1FAIpQLSfjiOnkbODuPifSGuzxYY61vB5kyMWdTZSSqkJsv3H6ePRTQA/viewform?usp=sf_link'
-                );
-              }}
-              primary
-              icon={<PublishIcon />}
-              label={<Trans>Submit your game to the showcase</Trans>}
-            />
-          ) : null,
-          this.state.currentTab === 'examples' ? (
-            <FlatButton
-              key="submit-example"
-              onClick={() => {
-                Window.openExternalURL(
-                  'https://github.com/GDevelopApp/GDevelop-examples/issues/new/choose'
-                );
-              }}
-              primary
-              icon={<PublishIcon />}
-              label={<Trans>Submit your project as an example</Trans>}
-            />
-          ) : null,
-        ]}
-        cannotBeDismissed={false}
-        onRequestClose={onClose}
-        open={open}
-        noMargin
-        fullHeight
-        flexBody
-      >
-        <Column expand noMargin>
-          <Tabs value={this.state.currentTab} onChange={this._onChangeTab}>
-            <Tab label={<Trans>Examples</Trans>} value="examples" />
-            <Tab label={<Trans>Tutorials</Trans>} value="tutorials" />
-            <Tab label={<Trans>Games showcase</Trans>} value="games-showcase" />
-          </Tabs>
-          {this.state.currentTab === 'examples' && (
-            <ExamplesComponent
-              onOpen={onOpen}
-              onChangeOutputPath={outputPath => this.setState({ outputPath })}
-              outputPath={this.state.outputPath}
-              onCreateFromExampleShortHeader={onCreateFromExampleShortHeader}
+  return (
+    <I18n>
+      {({ i18n }) => (
+        <>
+          <Dialog
+            title={<Trans>Create a new project</Trans>}
+            actions={actions}
+            secondaryActions={secondaryActions}
+            cannotBeDismissed={false}
+            onRequestClose={onClose}
+            open={open}
+            noMargin
+            fullHeight
+            flexBody
+          >
+            <Column expand noMargin>
+              <Tabs
+                value={currentTab}
+                onChange={(newTab: CreateProjectDialogTabs) => {
+                  setCurrentTab(newTab);
+                }}
+              >
+                <Tab label={<Trans>Examples</Trans>} value="examples" />
+                <Tab label={<Trans>Tutorials</Trans>} value="tutorials" />
+                <Tab
+                  label={<Trans>Games showcase</Trans>}
+                  value="games-showcase"
+                />
+              </Tabs>
+              {currentTab === 'examples' && (
+                <Column noMargin expand useFullHeight>
```

Comment:
why has this tab a column and the others not? Should this be part of the ExampleStore?

Reply:
Well, ExampleStore and CreateProjectDialog need the ProjectPreCreationDialog, so I figured it would be better having only one instance of it. But the logic and the component is highly similar to what is done in the homepage. It could even be refactored to be in a provider.